### PR TITLE
Recursive directory creation for target

### PR DESCRIPTION
**Pull request by mayo**
When copying a single file or nested directory, copy the asset recursively. For example:

```
src: asset/media/css/styles.css
dest: media/css/styles.css
```

would fail, because media/css directory doesn't exist within metalsmit's destination (or build) directory. Specifying a new `createDest: true` option, will create the required rectories recursively. `createDest` defaults to false, to preserve the original behaviour.

### CODE REVIEW

**Comment by thehydroimpulse** [comment]
@mayo Sorry for the late response, didn't see it in my notifications.

File: package.json [hunk 1]
```diff
@@ -1,12 +1,16 @@
 {
   "name": "metalsmith-static",
+  "repository": "git://github.com/mayo/metalsmith-static.git",
```
Comment (thehydroimpulse) [inline package.json]: Can you change this?

**Comment by thehydroimpulse** [comment]
Merged into 19772dd6d88cfd022de65fbfda03a53173c9068a

**Comment by thehydroimpulse** [comment]
Thanks!

**Comment by mayo** [comment]
No worries, thank you!

**Pull request closed without merging**